improved the background of the login page

aditya-bhaumik opened the pull request:
## Description

i have added hover effect to the social icons and the home button 
I have added a linear gradient background 
i have added a wave animation to the new linear background 

## Related Issues

[Cite any related issue(s) this pull request addresses. If none, simply state “None”]
- Closes #834 

## Type of PR

- [ ] Bug fix
- [ ] Feature enhancement
- [ ] Documentation update
- [ ] Other (specify): _______________

## Screenshots / videos (if applicable)
Before :

![image](https://github.com/user-attachments/assets/d08a08f2-5790-4997-a520-446938dbb074)

After : 

![image](https://github.com/user-attachments/assets/b0e6fa33-2d69-4193-8a21-4b2e252a78b7)

The background is continuously moving




## Checklist

- [ ] I have gone through the [contributing guide](https://github.com/Anishkagupta04/RAPIDOC-HEALTHCARE-WEBSITE-/)
- [ ] I have updated my branch and synced it with project `main` branch before making this PR
- [ ] I have performed a self-review of my code
- [ ] I have tested the changes thoroughly before submitting this pull request.
- [ ] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [ ] I have commented my code, particularly in hard-to-understand areas.
<!-- [X] - put a cross/X inside [] to check the box -->

## Additional context:
[Include any additional information or context that might be helpful for reviewers.]

aditya-bhaumik commented:
Please review my pull request @varshith257 @Anishkagupta04

varshith257 commented:
Hey @aditya-bhaumik , it's good with it's color scheme matching with project. Can you remove the animation and keep static color matching with color scheme ?

You can also make it mobile responsive it's looking really bad in mobile view of login/sign up page

aditya-bhaumik commented:
@varshith257 Sure, I’ll make the changes and let you know as soon as I can.

aditya-bhaumik commented:
@varshith257 i have made the background static and now the page is more responsive then before 
![image](https://github.com/user-attachments/assets/6f2ddf0c-dccc-4144-98fd-4925255540e9)

![image](https://github.com/user-attachments/assets/93fabf81-ac3a-49b5-b40c-05d58a890fc1)

![image](https://github.com/user-attachments/assets/fa9bd4e4-f076-40f5-8496-25d0ed89df91)

![image](https://github.com/user-attachments/assets/0682816d-f1c4-4e8c-b703-42e05f0dac1e)

varshith257 commented:
![IMG_20240714_000030.jpg](https://github.com/user-attachments/assets/1429d5e5-617c-45c0-917a-ae8414baf863)

See here some line is appearing in the middle of page . I think it is due to animation

varshith257 commented:
For mobile response I think we have to make text size small and same buttons and social media icons to better align with mobile screen

aditya-bhaumik commented:
@varshith257 now the line should not be coming i made the text and all small can you check again for different devices

aditya-bhaumik commented:
i checked in mine for different devices and it is looking good in all of them

![image](https://github.com/user-attachments/assets/0bc493b9-b678-4b00-9afc-59d36f62dd5f)
![image](https://github.com/user-attachments/assets/a61a8633-8ead-47cc-abcf-7f7762d7f290)
![image](https://github.com/user-attachments/assets/caf31e8f-d3d4-4821-a2f2-813969353f05)

varshith257 commented:
> i checked in mine for different devices and it is looking good in all of them
> 
> ![image](https://github.com/user-attachments/assets/0bc493b9-b678-4b00-9afc-59d36f62dd5f)

Exactly here I think we have to define multiple media screens if needed to inline the remembere and forgot password could align in one line

aditya-bhaumik commented:
@varshith257 hi i tried align the 2 text in same line but no matter how many media queries i add there are some screens where these 2 are not coming in same line

varshith257 commented:
@aditya-bhaumik Have you tried with reducing text sizes where isn't aligning

aditya-bhaumik commented:
@varshith257 yes it is just make text smaller it is not getting them in the same line

aditya-bhaumik commented:
![image](https://github.com/user-attachments/assets/1c55e1b1-7d9e-40c5-9019-237ff652d4cc)
see like this

varshith257 commented on log/login.css:
```diff
@@ -341,4 +342,112 @@ input {
+}
+
+/* Adjustments for screens smaller than 768px */
+@media (max-width: 768px) {
```
Use <a> and <label>  tags here to reduce their sizes as how remember me and forgot password are aligned in login.html

And also move the login.html page here to log folder 

login.html has multiple <script> tags (duplicate) remove them too

aditya-bhaumik commented:
@varshith257 i added the login.html to the log folder and removed the duplicate script code i also tried the labels you mentioned but the result is same as before

varshith257 commented:
Ok, let's come to it later. I have no time to see what's the cause of it

aditya-bhaumik commented:
@varshith257 There was just a small change left that I was making before the code got merged. In the index.html, the link was supposed to be log/login.html to link the login page can you please make that small change

varshith257 commented:
Submit follow up PR with this change

aditya-bhaumik commented:
@varshith257 added it in #855
